dropshot/src/router.rs

```diff
@@ -532,10 +532,21 @@ impl<Context: ServerContext> HttpRouter<Context> {
         if node.methods.values().any(|handlers| {
             find_handler_matching_version(handlers, version).is_some()
         }) {
-            Err(HttpError::for_client_error_with_status(
+            let mut err = HttpError::for_client_error_with_status(
                 None,
                 ClientErrorStatusCode::METHOD_NOT_ALLOWED,
-            ))
+            );
+
+            // Add `Allow` headers for the methods that *are* acceptable for
```

Comment:
cite the RFC?

Reply:
good call, done in https://github.com/oxidecomputer/dropshot/pull/1193/commits/82d4f91c05cfa111952f08099011684416d72939!